File: 30 Days of Code/Day 2- Operators/Operators.swift
import Foundation

let mealCost = Double( readLine() ?? "" ) ?? 0
let tip = Double(readLine() ?? "") ?? 0
let tax = Double(readLine() ?? "") ?? 0

var totalCost = Int(round(mealCost + ((tip/100) * mealCost) + ((tax/100) * mealCost)))

print("The total meal cost is \(totalCost) dollars.")

File: 30 Days of Code/Day 2- Operators/Operators.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {
    double mealCost;
    double tip;
    double tax;
    int totalCost;
    
    scanf("%lf", &mealCost);
    scanf("%lf", &tip);
    scanf("%lf", &tax);
    
    totalCost = round(mealCost + ((tip/100) * mealCost) + ((tax/100) * mealCost));
    
    printf("The total meal cost is %i dollars.", totalCost);
    
    return 0;
    
}

